*** libtool.orig	Thu Jun 20 16:01:33 2002
--- libtool	Thu Jun 20 17:33:38 2002
***************
*** 1853,1858 ****
--- 1853,1867 ----
  	*) . ./$lib ;;
  	esac
  
+ 	# remove anything from the dependency_libs value which references opengl libs
+ 	# this sed simply matches the pattern:
+ 	#     (zero-or-more-non-whitespace)libGL(zero-or-more-non-whitespace) 
+ 	# 	and
+ 	#     -lGL(zero-or-more-non-whitespace) 
+ 	# and replaces it with ''
+ 	dependency_libs=`$echo "$dependency_libs" | sed -e "s/[^[:space:]]*libGL[^[:space:]]*/ /g"`
+ 	dependency_libs=`$echo "$dependency_libs" | sed -e "s/-lGL[^[:space:]]*/ /g"`
+ 
  	if test "$linkmode,$pass" = "lib,link" ||
  	   test "$linkmode,$pass" = "prog,scan" ||
  	   { test "$linkmode" = oldlib && test "$linkmode" = obj; }; then
